src/pages/lib-v1/auth/getting-started/q/platform/[platform].mdx

```diff
@@ -124,6 +124,12 @@ The CLI prompts will help you to customize your auth flow for your app. With the
 - Customize attributes for your users, e.g. name, email
 - Enable 3rd party social providers, e.g. Facebook, Twitter, Google and Amazon
 
+<Callout warning>
+
+  If you need to support Multi-Factor Authentication (i.e. TOTP & SMS codes) in your application make sure to enable it during this step to avoid having to revisit this process.
+
```

Comment:
Should we link the [Set up Backend resources section](/lib-v1/auth/mfa/q/platform/js/#set-up-backend-resources) ?

Reply:
Done! Good catch